server/backends/blobstore/gcp/gcp.go

```diff
@@ -0,0 +1,136 @@
+package gcp
```

Comment:
ugh, sorry I missed this in the last review round. I think maybe this should be "gcs" to be consistent with the blobstore name. Otherwise LGTM

Reply:
auto-merge strikes again :P I can send a PR to take care of it

Reply:
oh that is a good point, I hadn't thought of that. I just named it based on the blobstore name, like `NewAWSBlobstore` etc